src/plugins/intel_cpu/src/cpu_tensor.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <memory>
+#include <string>
+#include <map>
+
+#include "cpu_tensor.h"
+#include "ie_blob.h"
+#include "ie_ngraph_utils.hpp"
+#include "openvino/runtime/iremote_tensor.hpp"
+#include "openvino/runtime/properties.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+//For the beginning probably one contructor should be enough
+Tensor::Tensor(MemoryPtr memptr) : m_memptr{memptr}, m_strides{}, need_update_strides{true} {
+    OPENVINO_ASSERT(m_memptr != nullptr);
+
+    // only support plain data format ncsp.
+    auto blocked_desc = m_memptr->getDescWithType<BlockedMemoryDesc>();
+    OPENVINO_ASSERT(blocked_desc, "intel_cpu::Tensor only support blocked memory.");
+    OPENVINO_ASSERT(blocked_desc->hasLayoutType(LayoutType::ncsp), "intel_cpu::Tensor only support memory with ncsp layout.");
+
+    m_element_type = InferenceEngine::details::convertPrecision(blocked_desc->getPrecision());
+    m_shape = blocked_desc->getShape().toPartialShape().to_shape();
+
+    if (!shape_size(m_shape)) m_memptr->getMemoryMngr()->resize(0);
```

Comment:
MemoryMngrWithReuse with resize(0) will do something actually. It will return a non-null ptr.
So is ov::Tensor data() with shape{0}.
Need to align with core team on the expected behavior.

Reply:
As there is still undecisive of the expected behavior, I will delete the corresponding unit test and the code change for it.